bindings/src/main/scala/chrome/contextMenus/bindings/ContextMenus.scala

```diff
@@ -0,0 +1,114 @@
+package chrome.contextMenus.bindings
+
+import chrome.tabs.bindings.Tab
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.{JSName, ScalaJSDefined}
+import scala.scalajs.js.|
+import scala.scalajs.js.JSConverters._
+
+object MenuContexts {
+
+  val ALL = "all"
+  val PAGE = "page"
+  val FRAME = "frame"
+  val SELECTION = "selection"
+  val LINKE = "link"
+  val EDITABLE = "editable"
+  val IMAGE = "image"
+  val VIDEO = "video"
+  val AUDIO = "audio"
+  val LAUNCHER = "launcher"
+  val BROWSER_ACTION = "browser_action"
+  val PAGE_ACTION = "page_action"
+}
+
+@js.native
+@JSName("chrome.contextMenus")
+object ContextMenus extends js.Object{
+
+  def create(createProperties: CreateProperties): String | Int = js.native
+
+  def update(id: String | Int, properties: UpdateProperties): Unit = js.native
+
+  def remove(menuItemId: String | Int, callback: js.Function0[Unit]): String | Int = js.native
+
+  def removeAll(callback: js.Function0[Unit]): Unit = js.native
+
+}
+
+@js.native
+@JSName("chrome.contextMenus.onClicked")
```

Comment:
this should not be needed. You can do this like shown here https://github.com/lucidd/scala-js-chrome/blob/master/bindings/src/main/scala/chrome/alarms/bindings/Alarms.scala#L13.